### ISSUE TITLE
False Positive: AI022700 (Microsoft WCF)

### ISSUE THREAD

**Issue by scovetta** [microsoft/ApplicationInspector#242]
Found a false positive on one of the rules:

```
        "pattern": "ServiceContract|using System\\.ServiceModel|WSHttpBinding|[HttpGet]|[HttpPut]",
```

The [HttpGet] and [HttpPut] strings need to have the brackets escaped, otherwise they're interpreted as regexs (any H, t, p, P, u, or t character, etc.).

**Comment by guyacosta** [comment]
Thanks!
